data_steward/tools/delete_stale_test_buckets.py

```diff
@@ -0,0 +1,59 @@
+"""
+Delete stale buckets in test environment. 
+"""
+from google.cloud import storage
+from datetime import datetime, timezone
+import logging
+
+# Project imports
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+CLIENT = storage.Client(project='aou-res-curation-test')
```

Comment:
You can import `from gcloud.gcs import StorageClient` which will be scoped to this project. 🙂

Reply:
Thanks, @MichaelSchmidt82 ! I wanted to avoid hardcoding the project, so I will try it 😃